administrator/language/de-DE/com_config.ini

```
@@ -20,17 +20,21 @@ COM_CONFIG_ERROR_CONFIG_EXTENSION_NOT_FOUND="Die Konfigurationserweiterung konnt
 COM_CONFIG_ERROR_CONFIGURATION_PHP_NOTUNWRITABLE="Der Schreibschutz der Datei „configuration.php“ konnte nicht entfernt werden."
 COM_CONFIG_ERROR_CONFIGURATION_PHP_NOTWRITABLE="Der Schreibschutz der Datei „configuration.php“ konnte nicht umgangen werden."
 COM_CONFIG_ERROR_CUSTOM_CACHE_PATH_NOTWRITABLE_USING_DEFAULT="Das Verzeichnis „%1$s“ ist schreibgeschützt und kann nicht für den Zwischenspeicher (Cache) verwendet werden. Es wird stattdessen das Standardverzeichnis „%2$s“ verwendet."
+COM_CONFIG_ERROR_CUSTOM_LOG_PATH_NOTWRITABLE_USING_DEFAULT="Der Ordner %1$s ist nicht beschreibbar und kann nicht für den Protokollordner verwendet werden. Bitte den Standardordner %2$s verwenden."
 COM_CONFIG_ERROR_CUSTOM_SESSION_FILESYSTEM_PATH_NOTWRITABLE_USING_DEFAULT="The folder at %s is not writable and cannot be used to store session data, the default PHP path will be used instead."
+COM_CONFIG_ERROR_CUSTOM_TEMP_PATH_NOTWRITABLE_USING_DEFAULT="Der Ordner %1$s ist nicht beschreibbar und kann nicht für den temporären Ordner verwendet werden, stattdessen wird der Standardordner %2$s verwendet."
```

```suggestion
COM_CONFIG_ERROR_CUSTOM_TEMP_PATH_NOTWRITABLE_USING_DEFAULT="Das Verzeichnis %1$s ist schreibgeschützt und kann nicht als temporäres Verzeichnis verwendet werden. Es wird stattdessen das Standardverzeichnis „%2$s“ verwendet."
```

See previous translations